remotespark/livyclientlib/livyunexpectederror.py

```diff
@@ -0,0 +1,3 @@
+class LivyUnexpectedError(Exception):
+    """An exception that will be shown if some unexpected error happens on the Livy side."""
```

Comment:
My idea was that this exception could be used for other uses besides just an unexpected final status but this is OK too.